std_msgs/String robot_name
int8 robot_state